src/System.Management.Automation/FormatAndOutput/common/TableWriter.cs

```diff
@@ -371,7 +371,7 @@ private StringCollection GenerateMultiLineRowField(string val, int k, int alignm
         {
             StringCollection sc = StringManipulationHelper.GenerateLines(dc, val,
                                         _si.columnInfo[k].width, _si.columnInfo[k].width);
-            if (addPadding || alignment == TextAlignment.Right)
+            if (addPadding || alignment != TextAlignment.Left)
```

Comment:
Maybe it makes sense to check explicit values - right and center - because there may be "width" alignment?

Reply:
Good point.  Will change.